scripts/github-release.js

```diff
@@ -0,0 +1,53 @@
+import {execSync} from 'child_process';

```

Comment:
Could these scripts go into ocular so they don't have to be copied into each repo?

Reply:
I prefer to keep it in each repo for now as we don't have standardized changelog format.